crates/dojo-types/src/primitive.rs

```diff
@@ -238,6 +272,10 @@ impl Primitive {
                 *value = Some(U256::from_be_bytes(bytes));
                 Ok(())
             }
+            // Primitive::I8(ref mut value) => {
```

Comment:
@glihm I commented the code I'm having issues here. I may not be doing it correctly so please have a look at it and let me know what needs to be done. Thanks!

Reply:
This change should actually be handled automatically with the upgrade to new `Felt` crate, depending on #2126.
Because with this one, we have conversions between `Felt` and signed integers iirc.

Reply:
So I don't need to include coverage for it in `deserialize` and `serialize` right?